dask/bag/core.py

```diff
@@ -846,6 +851,27 @@ def from_sequence(seq, partition_size=None, npartitions=None):
     return Bag(d, name, len(d))
 
 
+def from_url(urls):
+    """Create a dask.bag from a url
+
+    >>> a = from_url('http://google.com')
+    >>> a.npartitions
+    1
```

Comment:
Suggest pulling a small bit of text from a project gutenberg book or some website that is both identifiable and that we expect to be around for a while.  I've used "It was the best of times, It was the worst of times" from dickens a fair amount.  It's easily recognizable.